versioned_docs/version-2.0.0/keploy-cloud/installation.md

```diff
@@ -44,7 +44,7 @@ To get started with Keploy Cloud, you will need to install the Keploy Cloud Agen
 Install and Validate the Keploy Cloud Agent by following the steps below:
 
 ```bash
-curl --silent -O --location https://keploy-enterprise.s3.us-west-2.amazonaws.com/releases/latest/assets/keploy-enterprise.sh && source keploy-enterprise.sh
+curl --silent -O -L https://keploy.io/ent/install.sh && source install.sh
```

Comment:
<img width="1002" alt="image" src="https://github.com/keploy/docs/assets/53110238/7eaf4911-dfec-495b-8bec-8c8971e3c4c2">

Same issue with the enterprise one-click

Reply:
https://github.com/keploy/enterprise/pull/205